content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -254,15 +254,20 @@ Windows has strict compatibility rules, where the host OS version must match the
 * The existing node problem detector is Linux-only and requires privileged containers. In general, we don't expect this to be used on Windows because privileged containers are not supported
 * Not all features of shared namespaces are supported (see API section for more details)
 
-##### Memory Reservations and Handling
+##### Resource Reservations and Handling
 
+###### Memory Reservations
```

Comment:
Having H6 in the doc might be terrible. Please rethink if this is absolutely necessary.

Reply:
I am fine changing it, do you have any suggestions?

Reply:
How about a separate page in the reference section about Windows limitations & differences?

Reply:
@sftim Right now all of supported functionality, limitations, and troubleshooting are on the same page. Would should we split these out into 3 additional pages? I think it might not read a smoothly just moving limitations to separate page

Reply:
Maybe move troubleshooting to its own page then, inside https://kubernetes.io/docs/tasks/

Reply:
Do we have conclusion here? I am fine either way but I'd prefer limiting to a single page.

Reply:
@ravisantoshgudimetla troubleshooting a Windows node seems like a stand-alone task; do you think that split works?

Reply:
It might work but I am not in favor of that as part of this PR unless it is absolutely needed.

Reply:
OK. Another tack. Line 237 reads `#### Compute`. How about changing that heading to cover resources in general, and then the `###### CPU Reservations` heading need only be a level 5 heading?

This would also mean moving `##### Operating System Restrictions` later in the document.

Reply:
I am fine doing that, I'll create a separate PR for that.

@marosset WDYT?

Reply:
Sounds good to me.
Let me know if you need any help here.